libs/blocks/path-finder/path-finder.css

```diff
@@ -0,0 +1,91 @@
+:root {
+  /* Colors */
+  --hlx-sk-bg: rgb(48 48 48);
+  --hlx-sk-color: #FFF;
+  --notch-size: 12px;
```

Comment:
Should this also be scoped to `hlx` or sidekick or tools, etc? To avoid any clash and make its purpose clear.

Reply:
Updated.